### PR TITLE
Create pull request template

### PR DESCRIPTION
**Pull request by jmarolf**
Fixes https://github.com/dotnet/roslyn/issues/8806

Changes Proposed in this pull request:
- Add PULL_REQUEST_TEMPLATE.md file [see here](https://help.github.com/articles/creating-a-pull-request-template-for-your-repository/)

@dotnet/roslyn

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+- 
+- 
+
+Choose a team to tag for review:
+@dotnet/debugger-team 
+@dotnet/roslyn-analysis
+@dotnet/roslyn-compiler
+@dotnet/roslyn-ide
+
+Choose idividual to verify the fix:
```
Comment (jasonmalinowski) [inline .github/PULL_REQUEST_TEMPLATE.md]: Typo. Also, is this the pattern we want to be doing?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+Changes Proposed in this pull request:
+- 
+- 
+- 
+
+Choose a team to tag for review:
+@dotnet/debugger-team 
+@dotnet/roslyn-analysis
+@dotnet/roslyn-compiler
+@dotnet/roslyn-ide
```
Comment (jasonmalinowski) [inline .github/PULL_REQUEST_TEMPLATE.md]: Consider: infrastructure. Also why is debugger team not roslyn-debugger?

**Comment by jasonmalinowski** [comment]
Should this target stabilization so we get the template there too?

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (tmat) [inline .github/PULL_REQUEST_TEMPLATE.md]: interactive?

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (jmarolf) [inline .github/PULL_REQUEST_TEMPLATE.md]: I don't know this PR is here to start the conversation.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (jmarolf) [inline .github/PULL_REQUEST_TEMPLATE.md]: thanks, fixed

**Comment by jmarolf** [comment]
@jasonmalinowski I don't have the branch logic in my head.  Are we still submitting PRs against stabilization yet not merging anything in there in the near future? If so, I'll reopen this against stabilization

**Comment by jasonmalinowski** [comment]
@jmarolf: we are now submitting to stabilization again.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+Changes Proposed in this pull request:
+- 
+- 
+- 
+
+Choose a team to tag for review:
+@dotnet/roslyn-analysis
+@dotnet/roslyn-compiler
+@dotnet/roslyn-ide
+@dotnet/roslyn-interactive
```
Comment (jasonmalinowski) [inline .github/PULL_REQUEST_TEMPLATE.md]: `@dotnet/roslyn-infrastructure`?

**Comment by jmarolf** [comment]
@jasonmalinowski  Alright opening this against stabilization

**Pull request closed without merging**